vendor/k8s.io/kubernetes/pkg/volume/glusterfs/glusterfs.go

```diff
@@ -476,9 +484,9 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 		return nil, err
 	}
 	r.provisioningConfig = *cfg
-
 	glog.V(4).Infof("glusterfs: creating volume with configuration %+v", r.provisioningConfig)
-	glusterfs, sizeGB, err := r.CreateVolume()
+	reqGid = gidMin + gidRandomizer.Int63n(gidMax)
```

Comment:
@eparis the gidRandomizer https://github.com/openshift/origin/pull/11893/files#diff-1e1a6d9a7a87786923be6cfc3dfbc37fR475 , Isnt it ? The GID has been passed by https://github.com/openshift/origin/pull/11893/files#diff-1e1a6d9a7a87786923be6cfc3dfbc37fR502 as mentioned here  https://github.com/kubernetes/kubernetes/pull/35460#issuecomment-256951507